src/main/java/org/gelecekbilimde/scienceplatform/auth/model/request/RegisterRequest.java

```diff
@@ -28,6 +29,11 @@ public class RegisterRequest {
 	@Size(min = 2, max = 25)
 	private String lastname;
 
+	@NotBlank
+	@Username
+	@Size(min = 3, max = 20)
```

Comment:
Username validasyonumuz varsa karakter sınırını da orada belirtmemiz gerekir diye düşünüyorum. Dışarıya yazmak çok doğru bir yöntem değil

Reply:
@agitrubard  Size kısmında hata aldığımızda bu şekilde daha doğru bir mesaj geliyor diye böyle düzelttim.
Tekrardan validasyon içerisinde mi belirtmeliyim

Reply:
Evet, çünkü username'in size validasyonu nerede olursa olsun aynı olacak